### DISCUSSION TITLE
what should I do if I want to change the color and font size when md2word

### DISCUSSION THREAD

**Discussion by xwxb** [jgm/pandoc#9719] [Q&A] [answered]
Originally the problem I encountered was that using the `--reference-doc` option had no effect. Then I searched for the reference https://github.com/jgm/pandoc/issues/5974
But I still don’t quite understand it. If I understand it correctly, this means that styles such as font size and color are not styles that users can configure in the template.
So I want to understand directly, if I want to change the converted font size and text color (not the default blue), how should I set it up?

**Comment by jgm** [comment, marked as answer]
You can adjust font size and text color.  First get a copy of the default reference.docx:
```
$ pandoc --print-default-data-file reference.docx > ref.docx
```
Now open this in Word.  Go to Format->Styles and modify the "Body Text" style.  Change the font and Apply. You should see the font in the sample document change.  Now save this document, and use it in future pandoc conversions with `pandoc --reference-doc ref.docx`.

To change the heading colors and font, modify "Heading 1" etc.

**Reply by xwxb** [reply]
I finally got it to work, I didn't used to realize I should have changed the format from a more essential place.


Incidentally, my word was changed in this place, there seems to be no “format” entry, I changed it from here, for posterity!
<img width="417" alt="ECD646AF-2CAA-4128-8A36-98B3264F94C9" src="https://github.com/jgm/pandoc/assets/77780051/64ca6614-6e38-46b5-a320-783470261d6c">

The other thing is that using `>` for template generation doesn't seem to open the file correctly in powershell, using the official `-o` method is fine.

The last thing is that I also realized in retrospect that my question was not very clear, so I apologize for that. Thx for detailed reply🥰
